--- lighttpd.conf.orig	2014-08-24 17:08:31.872098914 +0200
+++ lighttpd.conf	2014-08-24 17:07:42.428100982 +0200
@@ -3,6 +3,7 @@
 	"mod_alias",
 	"mod_compress",
  	"mod_redirect",
+ 	"mod_auth",
 #       "mod_rewrite",
 )
 
@@ -14,6 +15,18 @@
 server.groupname            = "www-data"
 server.port                 = 80
 
+accesslog.filename          = "/var/log/lighttpd/access.log"
+auth.debug                  = 2
+auth.backend                = "plain"
+auth.backend.plain.userfile = "/home/pi/antifurto/config/.lighttpdpassword"
+auth.require                = ( "/" =>
+(
+    "method"  => "basic",
+    "realm"   => "Password protected area",
+    "require" => "user=admin"
+)
+)
+
 
 index-file.names            = ( "index.php", "index.html", "index.lighttpd.html" )
 url.access-deny             = ( "~", ".inc" )
@@ -27,3 +40,17 @@
 #include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
 include_shell "/usr/share/lighttpd/create-mime.assign.pl"
 include_shell "/usr/share/lighttpd/include-conf-enabled.pl"
+
+fastcgi.server += ( "/live/" =>
+((
+    "socket" => "/var/run/lighttpd/live.socket",
+    "bin-path" => "/var/www/cgi-bin/live.py",
+    "max-procs" => 1,
+    "bin-environment" => (
+        "REAL_SCRIPT_NAME" => ""
+    ),
+    "bin-copy-environment" => (
+        "PATH", "SHELL", "USER"
+    ),
+))
+)
